release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -487,13 +487,13 @@ For more information see, xref:../scalability_and_performance/routing-optimizati
 === Authentication and authorization
 
 [id="ocp-4-11-auth-oidc-providers"]
-==== Active Directory Federation Services is now a supported OIDC provider
+==== Additional supported OIDC providers
 
-The Active Directory Federation Services OpenID Connect (OIDC) provider is now tested and supported with {product-title}.
+The Active Directory Federation Services for Windows Server and Azure Active Directory OpenID Connect (OIDC) providers are now tested and supported with {product-title}.
```

Comment:
After reading https://docs.microsoft.com/en-us/azure/active-directory/azuread-dev/azure-ad-endpoint-comparison , got to know AAD has v1.0 and v2.0, I tested v2.0, maybe you can say: `and Azure Active Directory (v2.0)` or `and Microsoft identity platform` or fully `and Microsoft identity platform (Azure Active Directory v2.0)`

Reply:
For this one, maybe you also need to add one note due to it only syncs group IDs instead of group names: `Currently, it is not supported to use Microsoft identity platform when group names are required to be synced`